HomeWorkAssigment/src/Main.java

```diff
@@ -0,0 +1,281 @@
+import java.util.*;
+
+public class Main {
+   
+        // Списки
+
+   /*1. Есть два класса: Address с полями улица и номер дома и
+        Person с полями имя и Address. Нужно написать функцию:
+
+        List<Address> getAddresses(List<Person> persons)
+
+        то есть по списку persons возвращать список их адресов. */
+        public static class Address {
+            String street;
+            int num;
+
+            public Address(String street, int num) {
+                this.street = street;
+                this.num = num;
+
+            }
+
+            @Override
+            public String toString() {
+                return "Address{" +
+                        "street='" + street + '\'' +
+                        ", num=" + num +
+                        '}';
+            }
+        }
+
+        public static class Person {
+            String name;
+            String Address;
+
+            public Person(String name, String Address) {
+                this.name = name;
+                this.Address = Address;
+            }
+
+            public List<String> getAddress(List<Person> persons) {
+                List<String> list = new LinkedList<>();
```

Comment:
This method should not be in this class because it does nothing with the fields of the class (name and address). It should either be static and stay in this class or moved to a special class.